frontend/www/js/omegaup/problem/collection.ts

```diff
@@ -15,6 +16,16 @@ OmegaUp.on('ready', () => {
         props: {
           levelTags: payload.levelTags,
           problemCount: payload.problemCount,
+          wizardTags: payload.tagData,
+        },
+        on: {
+          'wizard-search': (queryParameters: {
+            [key: string]: string;
+          }): void => {
```

Comment:
```suggestion
          'wizard-search': (queryParameters: omegaup.QueryParameters): void => {
```

aquí es mejor utilizar el mismo tipo.

Reply:
me marca ` Argument of type 'QueryParameters' is not assignable to parameter of type '{ [key: string]: string; }'.
  Index signature is missing in type 'QueryParameters'`    en    `window.location.replace( /problem/?${ui.buildURLQuery(queryParameters)},);` , se necesita modificar también esa parte?

Reply:
ah ahí más bien hay que cambiar L26 por algo así como

```typescript
            window.location.replace(
              `/problem/?${ui.buildURLQuery(<{[key:string]: any}>queryParameters)}`,
            );
```

y cambiar `buildURLQuery()` en `frontend/www/js/omegaup/ui.ts` para que reciba `queryParameters: { [key: string]: any }`